config/rbac/osbuildconfig_editor_role.yaml

```diff
@@ -0,0 +1,24 @@
+# permissions for end users to edit osbuildconfigs.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: osbuildconfig-editor-role
```

Comment:
looks more like the admin role to me. Is there such thing?

Reply:
This is the names generated by the Operator SDK. I guess the template is `<CRD Name>-editor-role`